package.json

```diff
@@ -43,10 +43,8 @@
     "eslint": "eslint --color -c .eslintrc.js --ext js,jsx --max-warnings 0 .",
     "eslint-fix": "eslint --color -c .eslintrc.js --ext js,jsx --fix .",
     "eslint-fix-single": "eslint --color -c .eslintrc.js --fix",
-    "jest-cov": "jest --coverage --colors --silent --runInBand",
-    "jest-ci": "jest  --colors --silent --ci --runInBand --coverage --reporters=default --reporters=jest-junit && cat ./coverage/lcov.info | coveralls",
     "start": "npx babel-node server.js",
-    "test": "jest --colors --silent --detectOpenHandles",
+    "test": "jest --colors --silent --runInBand",
```

Comment:
Why this switch?

Reply:
detectOpenHandles is series, runInBand is parallel, as I recall.   runInBand seems to come along for the ride with npm test in other projects, I think???

Reply:
See https://jestjs.io/docs/cli#--runinband

Reply:
reverted this change.

Reply:
hah:  https://jestjs.io/docs/cli#--detectopenhandles   implies runInBand.  snort.

Reply:
so we need runInBand to keep memory needs down.  :-P